### ISSUE TITLE
nbcsports issue

### ISSUE THREAD

**Issue by johny1** [ytdl-org/youtube-dl#6502]
Hello. NBCsports is listed as a supported site. When I try to download a file, such as the following 

http://stream.nbcsports.com/nascar/?pid=18366

This comes up,

[generic] index_nbcsn-generic: Requesting header
WARNING: Falling back on generic information extractor.
[generic] index_nbcsn-generic: Downloading webpage
[generic] index_nbcsn-generic: Extracting information
ERROR: Unsupported URL: http://stream.nbcsports.com/nbcsn/index_nbcsn-generic.ht
ml?pid=18366

Is it maybe because I should be entering a username and password too when I run the command? I have one but I'm not sure if that's what causing this.

**Comment by dstftw** [comment]
Live extra streams are not supported.

**Comment by johny1** [comment]
The replays are also not supported?

**Comment by dstftw** [comment]
Post [an example URL](https://github.com/rg3/youtube-dl/blob/master/README.md#is-the-description-of-the-issue-itself-sufficient).

**Comment by johny1** [comment]
http://stream.nbcsports.com/liveextra/#replays

Then click on any stream listed under Replays, such as

http://stream.nbcsports.com/nascar/?pid=18390

**Comment by dstftw** [comment]
No. That's the same streaming delivery that requires cable TV provider.

**Comment by johny1** [comment]
I do have a provider login and pw and I am able to see the clips. 

But just to be confirm with what you stated, it still wouldn't work correct? Thanks.

**Comment by dstftw** [comment]
As said, this is not implemented.

**Comment by ts** [comment]
Ran into this earlier today. Unfortunate that it's being ignored. Any technical reasons preventing this or is it simply misunderstood?

URL: http://stream.nbcsports.com/gold/tour-de-france-stage-17
Cable provider: none; never had one, actually

**Comment by paleodictyon** [comment]
Sorry to jump onto an old issue, but are there any tricks to get NBC Sports Gold events to download?  I have a paid subscription to NBC SG, but can't seem to figure it out even with cable authentication and passing cookies from the browser.
